lte/gateway/c/core/oai/tasks/nas/emm/emm_main.c

```diff
@@ -62,8 +63,7 @@ void emm_main_initialize(const mme_config_t* mme_config_p) {
    */
   memset(&_emm_data.conf, 0, sizeof(_emm_data.conf));
   if (mme_api_get_emm_config(&_emm_data.conf, mme_config_p) != RETURNok) {
-    OAILOG_ERROR(
-        LOG_NAS_EMM, "EMM-MAIN  - Failed to get MME configuration data");
+    Fatal("EMM-MAIN  - Failed to get MME configuration data");
```

Comment:
Maybe add more context to message, e.g., "EMM-MAIN - Failed to get all required MME config data, check the error logs for missing configs"